base.css

```diff
@@ -94,6 +103,24 @@ button {
   align-items: flex-start;
 }
 
+.ali-ce {
+  align-items: center;
+}
```

Comment:
Naming can be improved. `y-center` would make sense , else `align-center`

Reply:
As there are 3-4 categories of align-items.. to make sure i understand each of them .. i followed this naming convention

Reply:
As there are 3-4 categories of align-items.. to make sure i understand each of them .. i followed this naming convention